pkg/sql/virtual_schema.go

```diff
@@ -734,9 +771,13 @@ func (vs *VirtualSchemaHolder) getVirtualTableEntry(tn *tree.TableName) (*virtua
 			return t, nil
 		}
 		if _, ok := db.allTableNames[tableName]; ok {
-			return nil, unimplemented.NewWithIssueDetailf(8675,
-				tn.Schema()+"."+tableName,
-				"virtual schema table not implemented: %s.%s", tn.Schema(), tableName)
+			return nil, unimplemented.NewWithIssueDetailf(
+				8975,
```

Comment:
I think this should still be 8675 as the issue #

Reply:
Nice catch!